monkey/monkey_island/cc/server_setup.py

```diff
@@ -60,6 +92,8 @@
         logger.warning("Setup only flag passed. Exiting.")
         return
 
+    bootloader_server_thread = _start_bootloader_server(MONGO_URL)
```

Comment:
Turns out `MONGO_URL` is unused in `BootloaderHttpServer.__init__`. It's best to move this out of this method, since it has nothing to do with starting the island.

Reply:
What's the purpose of the bootloader server? Is it not part of Monkey Island?

Reply:
The bootloader server is created where it is to avoid a bug that prevents Monkey Island from stopping when the `--setup-only` flag is passed.

@VakarisZ and I discussed that the `--setup-only` flag may be able to be removed after we separate user credentials from the `server_config.json`. If `--setup-only` goes away, then the bootloader sever thread can be managed elsewhere. In addition, we're not sure why we need the `BootloaderHttpServer`. The bootloaders could potentially just call API endpoints on Monkey Island. If the `BootloaderHttpServer` can be replaced, this becomes a non-issue. In either case, it's not worth trying to move this code now.

`MONGO_URL` will be removed from the `BootloaderHttpServer` constructor.